src/dataDisplay/personCoreMilestones/personCoreMilestones.jsx

```diff
@@ -1,165 +1,160 @@
 import {
-    includes,
+    includes, isEmpty,
```

Comment:
Should these be on separate lines?
_Not a Blocker_ but my understanding is we prefer it with each thing on separate lines, ever if there are only two or three, right?

Reply:
Fixed.  Resolving.